packages/core/src/components/ChoiceList/Choice.scss

```diff
@@ -57,22 +60,22 @@ Markup:
     <legend class="ds-c-label">Historical figures</legend>
     <span class="ds-c-field__hint ds-c-field__hint--inverse">Select all that apply</span>
     <input class="ds-c-choice ds-c-choice--inverse" id="truth-2" type="checkbox" name="historical-figures-2" value="truth" checked>
-    <label for="truth-2">Sojourner Truth</label>
+    <label for="truth-2" class="ds-u-flex-wrap--wrap ds-u-align-items--center">Sojourner Truth</label>
     <input class="ds-c-choice ds-c-choice--inverse" id="douglass-2" type="checkbox" name="historical-figures-2" value="douglass">
-    <label for="douglass-2">Frederick Douglass</label>
+    <label for="douglass-2" class="ds-u-flex-wrap--wrap ds-u-align-items--center">Frederick Douglass</label>
     <input class="ds-c-choice ds-c-choice--inverse" id="washington-2" type="checkbox" name="historical-figures-2" value="washington">
-    <label for="washington-2">Booker T. Washington</label>
+    <label for="washington-2" class="ds-u-flex-wrap--wrap ds-u-align-items--center">Booker T. Washington</label>
     <input class="ds-c-choice ds-c-choice--inverse" id="carver-2" type="checkbox" name="historical-figures-2" disabled>
-    <label for="carver-2">George Washington Carver</label>
+    <label for="carver-2" class="ds-u-flex-wrap--wrap ds-u-align-items--center">George Washington Carver</label>
   </fieldset>
   <fieldset class="ds-c-fieldset">
     <legend class="ds-c-label">Select a historical figure</legend>
     <input class="ds-c-choice ds-c-choice--inverse" id="truth-4" type="radio" name="historical-figures-4" value="truth" checked>
-    <label for="truth-4">Sojourner Truth</label>
+    <label for="truth-4" class="ds-u-flex-wrap--wrap ds-u-align-items--center">Sojourner Truth</label>
     <input class="ds-c-choice ds-c-choice--inverse" id="douglass-4" type="radio" name="historical-figures-4" value="douglass">
-    <label for="douglass-4">Frederick Douglass</label>
+    <label for="douglass-4" class="ds-u-flex-wrap--wrap ds-u-align-items--center">Frederick Douglass</label>
     <input class="ds-c-choice ds-c-choice--inverse" id="washington-4" type="radio" name="historical-figures-4" value="washington">
-    <label for="washington-4">Booker T. Washington</label>
+    <label for="washington-4" class="ds-u-flex-wrap--wrap ds-u-align-items--center">Booker T. Washington</label>
```

Comment:
Since we have a style declaration block for this element (`.ds-c-choice + label`) could we include these styles there to simplify implementation?